client/src/visualiser-src/sorting-visualiser/data-structure/GraphicalSorts.ts

```diff
@@ -33,6 +37,27 @@ export default class GraphicalSortList extends GraphicalDataStructure {
     return producer;
   }
 
+  public delete(values: number[]): AnimationProducer {
+    const producer = new SortsCreateAnimationProducer();
+    // Create a new list of values
+    const listValues = this.elementList.map((element) => element.data.value);
+    // Empty current list: values will be re-inserted
+    this.elementList = [];
+
+    // Remove each value the user inputs from the list
+    values.forEach((value) => {
```

Comment:
https://stackoverflow.com/questions/1723168/what-is-the-fastest-or-most-elegant-way-to-compute-a-set-difference-using-javasc

The accepted answer has a nice way of subtracting one list from another in JavaScript 😎